File: app/src/main/java/com/syber/ssspltd/response/CreditNoteReportRespo/CreditNoteReportResult.java

package com.syber.ssspltd.response.CreditNoteReportRespo;

import java.util.List;

import com.google.gson.annotations.SerializedName;
@SuppressWarnings("unused")
public class CreditNoteReportResult {

    @SerializedName("BillNo")
    private String mBillNo;
    @SerializedName("Date")
    private String mDate;
    @SerializedName("ItemsDetailsData")
    private List<ItemsDetailsDatum> mItemsDetailsData;
    @SerializedName("PDFPath")
    private String mPDFPath;
    @SerializedName("SRNO")
    private String mSRNO;
    @SerializedName("SaleBillDate")
    private String mSaleBillDate;
    @SerializedName("SaleBillNo")
    private String mSaleBillNo;
    @SerializedName("SupplierName")
    private String mSupplierName;

    @SerializedName("NetAmt")
    private String mNetAmt;



    public String getBillNo() {
        return mBillNo;
    }

    public void setBillNo(String billNo) {
        mBillNo = billNo;
    }

    public String getDate() {
        return mDate;
    }

    public void setDate(String date) {
        mDate = date;
    }

    public List<ItemsDetailsDatum> getItemsDetailsData() {
        return mItemsDetailsData;
    }

    public void setItemsDetailsData(List<ItemsDetailsDatum> itemsDetailsData) {
        mItemsDetailsData = itemsDetailsData;
    }

    public String getPDFPath() {
        return mPDFPath;
    }

    public void setPDFPath(String pDFPath) {
        mPDFPath = pDFPath;
    }

    public String getSRNO() {
        return mSRNO;
    }

    public void setSRNO(String sRNO) {
        mSRNO = sRNO;
    }

    public String getSaleBillDate() {
        return mSaleBillDate;
    }

    public void setSaleBillDate(String saleBillDate) {
        mSaleBillDate = saleBillDate;
    }

    public String getSaleBillNo() {
        return mSaleBillNo;
    }

    public void setSaleBillNo(String saleBillNo) {
        mSaleBillNo = saleBillNo;
    }

    public String getSupplierName() {
        return mSupplierName;
    }

    public void setSupplierName(String supplierName) {
        mSupplierName = supplierName;
    }
    public String getNetAmt() {
        return mNetAmt;
    }

    public void setNetAmt(String mNetAmt) {
        this.mNetAmt = mNetAmt;
    }

}

File: app/src/main/java/com/syber/ssspltd/data/model/CreditNotTo/CreditNoteReportResult.kt
package com.syber.ssspltd.data.model.CreditNotTo

import com.google.gson.annotations.SerializedName
import com.syber.ssspltd.data.model.debitNoteToCustomer.ItemsDetailsDatum

data class CreditNoteReportResult(

    @SerializedName("BillNo")
    val billNo: String? = null,

    @SerializedName("Date")
    val date: String? = null,

    @SerializedName("ItemsDetailsData")
    val itemsDetailsData: List<ItemsDetailsDatum>? = emptyList(),

    @SerializedName("PDFPath")
    val pdfPath: String? = null,

    @SerializedName("SRNO")
    val srno: String? = null,

    @SerializedName("SaleBillDate")
    val saleBillDate: String? = null,

    @SerializedName("SaleBillNo")
    val saleBillNo: String? = null,

    @SerializedName("SupplierName")
    val supplierName: String? = null,

    @SerializedName("NetAmt")
    val netAmt: String? = null
)

